docker-compose.yml

```diff
@@ -47,7 +45,7 @@ services:
       SUPPORT_URL: 'https://www.example.org/support'
       SUPPORT_FEEDBACK: 'https://www.example.org/support-feedback'
       UI_URL: 'https://profile.gtis.guru/#'
-      LOGO_URL: 'https://www.sil.org'
+      LOGO_URL: 'https://www.example.org/logo.png'
```

Comment:
I was about to suggest using dummyimage.com to create a real image, but in the process I discovered that LOGO_URL isn't even a valid variable for idp-pw-api.
```suggestion
```

Reply:
Good catch. Thanks.